src/core/gosling.schema.ts

```diff
@@ -366,6 +353,25 @@ export type OverlaidTrack = Partial<SingleTrack> &
         overlay: Partial<Omit<SingleTrack, 'height' | 'width' | 'layout' | 'title' | 'subtitle'>>[];
     };
 
+/*
+ * The styles of the effects of mouse events, such as mouse over on marks.
+ */
+export interface EventStyle {
+    color?: string;
+    stroke?: string;
+    strokeWidth?: number;
+    strokeOpacity?: number;
+    opacity?: number;
+}
+
+/*
+ * Show event effects behind or in front of marks.
+ * __Default__: `'front'`
+ */
+export interface EventArrange {
+    arrange?: 'behind' | 'front';
+}
+
```

Comment:
Could just unify with 

```typescript
export interface EventStyle {
    color?: string;
    stroke?: string;
    strokeWidth?: number;
    strokeOpacity?: number;
    opacity?: number;
    arrange?: 'behind' | 'front',
}
```

And then later,

```typescript
  brush: Omit<EventStyle, 'arrange'>,
```

for interface which don't have `arrange`

Reply:
oh oops, just seeing you merged. no worries!

Reply:
Oh, actually this is better.